examples/complex/esp32/ios-gateway/extras/iOS_IoTCore_Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,79 @@
+{
```

Comment:
I think you want to remove this file, it's calculated every time you run `pod update; pod install`